### ISSUE TITLE
Google Chrome Circle Font Rendering

### ISSUE THREAD

**Issue by developer-don** [drinchev/monosocialiconsfont#1]
Using chrome 26.0.1410.64 m. No Add ons or apps installed.
several circle... icons don't render. They do render in Mozilla.

**Comment by drinchev** [comment]
Would you mind, posting your OS too, along with a screenshot of the demo page ( http://drinchev.github.io/monosocialiconsfont/ ). Thanks

**Comment by developer-don** [comment]
![untitled-1](https://f.cloud.github.com/assets/3324653/462643/18e42220-b4cd-11e2-96e2-22fe4863423e.jpg)
Windows 7

**Comment by drinchev** [comment]
Thanks, I think it's an issue with chrome itself, I will try to make a bugfix this week.

**Comment by drinchev** [comment]
Workaround :

You can use the unicode escape character for those icons, until I find what's wrong with the ligature feature in google chrome for those. ( e.g. : just replace `circledeviantart` with `&#xe218` )

**Comment by drinchev** [comment]
I've reported a [bug](https://code.google.com/p/chromium/issues/detail?id=244438&q=ligatures&colspec=ID%20Pri%20M%20Iteration%20ReleaseBlock%20Cr%20Status%20Owner%20Summary%20OS%20Modified) to Chrome

**Comment by developer-don** [comment]
Cool,
I like the icon font. I'd like to see Chrome patch this bug.

**Comment by mintbird** [comment]
Access your demo site with latest opera (12.15) can't see any icons too.
It's looks like the same problem as Entypo Font. Remapping with lower character code may solve this problem.

**Comment by drinchev** [comment]
Yes. Opera doesn't support font-feature so you have to use Unicode characters there.

e.g. instead of `facebook` write `&#xe027;`.

I'm creating a script for those browsers. Stay tuned.
